### ISSUE TITLE
Removes styling when restructure is set to true

### ISSUE THREAD

**Issue by bjarnef** [ben-eb/gulp-csso#25]
It seems the gulp plugin removes styling from css when `restructure: true`

In Bootstrap 3.3.5+ the following has been adding to dropdowns.less

```
// Dropdown arrow/caret
.caret {
  display: inline-block;
  width: 0;
  height: 0;
  margin-left: 2px;
  vertical-align: middle;
  border-top:   @caret-width-base dashed;
  border-top:   @caret-width-base solid ~"\9"; // IE8
  border-right: @caret-width-base solid transparent;
  border-left:  @caret-width-base solid transparent;
}
```
https://fossies.org/diffs/bootstrap/3.3.4_vs_3.3.5/less/dropdowns.less-diff.html

However when csso is enabled or `restructure: true` it removes the first `border-top` line from output, e.g.
```
border-top: 4px solid\9;
```
but is should be:
```
border-top: 4px dashed;
border-top: 4px solid\9;
```
It has also been disqussed in csso here https://github.com/css/csso/issues/231
but it should have been fixed here https://github.com/css/csso/commit/87c715836c61980167ea67413717fd436ed93801

and marked as fixed in v. 1.4.0
https://github.com/css/csso/releases/tag/v1.4.0 (see *don't merge \9 hack values (#231)*)

but it doesn't work in latest version of gulp-csso (2.0.0).
not sure if something isn't included from csso?

**Comment by bjarnef** [comment]
Not sure if it is just an issue when using `\9`, because a common way, when one add same css property name twice is for fallback values, e.g. fallback color when `rgba` not is supported, where you don't want it to remove first line:
```
background: #f00;
background: rgba(255, 0, 0, 0.5);
```
https://css-tricks.com/rgba-browser-support/

**Comment by ben-eb** [comment]
I'm not excluding anything from CSSO, perhaps you have encountered a regression?

/cc @lahmatiy

**Comment by lahmatiy** [comment]
@bjarnef Looks like a bug:

```css
.not-ok {  /* your case*/
  border-top: 4px dashed;  /* will to be removed */
  border-top: 4px solid\9;
}
.ok {
  border-top: 4px dashed;
  border-top: 4px solid \9; /* extra space fixes issue */
}
```

There is no issue with `rgba()` try here http://css.github.io/csso/csso.html

Could you fill an issue in [csso bug tracker](https://github.com/css/csso/issues)?

**Comment by bjarnef** [comment]
okay, but it does remove `border-top: 4px dashed;` from output css, when I add gulp-csso to the task. When I remove it from the task or disable `restructure` it is working fine. So something in `gulp-csso` or `csso`.

I have the following in my task and csso saves a few kilobytes and each bundle.
However if I add `gulp-clean-css` instead of csso it returns ~same sizes as with csso (with restructure enabled). Without gulp-clean-css or when csso (restructured disabled) each bundle is a few kilobytes larges:

bootstrap.min.css:
112 KB (restructure: false)
110 KB (restructure: true)

```
.pipe(plugins.cssnano({
                discardComments: { removeAll: true },
                discardDuplicates: true,
                mergeLonghand: true,
                mergeRules: true,
                colormin: false, // (default: true) when setting to false rgba color values are not transformed to hsla
                zindex: false
}))
.pipe(plugins.cleanCss())
//.pipe(plugins.csso({
     //    restructure: false //issue when true, e.g. for .caret in Bootstrap where "\9" is used.
//}))
```

**Comment by lahmatiy** [comment]
@bjarnef Thank you!

**Comment by lahmatiy** [comment]
Fixed in CSSO 2.1.1
